### DISCUSSION TITLE
Pimcore Academy: Error when try update from 11.1.1 to 11.1.2

### DISCUSSION THREAD

**Discussion by chrobaks** [pimcore/pimcore#16322] [General]
Hi, i'm working with Linux/Ubuntu and do the Pimcore Academy Junior with the docker-imgae from Pimcore. I'll try to make the update from version 11.1.1 to 11.1.2 with these commands:
- docker compose exec php composer update
- docker compose exec php bin/console pimcore:cache:clear
- docker compose exec php bin/console doctrine:migrations:migrate --prefix=Pimcore\\Bundle\\CoreBundle
And than i got this error because in documents_page the column metaData not exsits:

[error] Migration Pimcore\Bundle\CoreBundle\Migrations\Version20210428145320 failed during Execution. Error: "An exception occurred while executing a query: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'metaData' in 'documents_page'"

I try a quick&dirty and add this column to the table, but than:

[error] Migration Pimcore\Bundle\CoreBundle\Migrations\Version20211018104331 failed during Execution. Error: "There is no column with name "o_id" on table "object_classificationstore_groups_AP"."

So somethings is bad with my database and i don't know what.
Are there any solutions?

**Comment by kingjia90** [comment]
Not sure, but there were no migrations from `11.1.1` to `11.1.2` so it's not related to that.  

Might be something that wasn't marked as done got triggered (see datetime in name `Version20210428145320`), maybe something got cleared in `migrations_versions` table in the DB?

**Reply by chrobaks** [reply]
Thanks for answer, is there any strategies to find out if it is a migration problem?
